### DISCUSSION TITLE
Mar 4th week sprint

### DISCUSSION THREAD

**Discussion by zangsu** [zangsu/cs-test-service#19] [Sprint]
코드 프리징까지 1주일 남았습니다!
3월 넷째주 개인 과업 목표를 설정해서 올려주세요!

**Comment by zangsu** [comment]
# 백엔드 주간 목표
+ `/user`, `/result`에 사용되는 컨트롤러 구현
+ 테스트 코드 작성
+ 디버깅 및 서비스 배포 준비
+ CS 문제 만들기

# 일간 목표
+ `/user` 컨트롤러 구현 : ~ 3/20 (월)
+ `/result` 컨트롤러 구현 : ~ 21 (화)
+ 테스트 코드 작성, 문제 작성 : ~ 22 (수) 
+ 서비스 배포 준비 : ~ 25 (토)

**Comment by Moon-GD** [comment]
# 프런트엔드 주간 목표
- 문제 화면 통신 연결
- 결과 화면 마크업 완성
- 결과 화면 통신 연결

# 일간 목표
- 문제 화면 통신 연결 : ~ 3 / 21 (화)
- 결과 화면 마크업 완성 : ~ 3 / 23 (목)
- 결과 화면 통신 연결 : ~ 3 / 25 (토)

**Comment by Moon-GD** [comment]
# 공통 주간 목표
- 3월 26일 (일) 배포 전 코드 최종 점검
